src/main/java/org/gridsuite/modification/server/dto/BatteryCreationInfos.java

```diff
@@ -68,15 +67,6 @@ public class BatteryCreationInfos extends InjectionCreationInfos {
     @Schema(description = "Reactive capability curve")
     private Boolean reactiveCapabilityCurve;
 
-    @Schema(description = "Connection Name")
```

Comment:
This move of connectionName, connectionDirection and connectionPosition fields up to InjectionCreationInfos class does not really concern the reactive limits refactoring ...
Same remark for the same move in the other dto classes and for the same move in the entity classes ...